civihr_employee_portal/views/templates/_views-view--generic-display-output.tpl.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * @file
+ * Main view template.
+ *
+ * Variables available:
+ * - $classes_array: An array of classes determined in
+ *   template_preprocess_views_view(). Default classes are:
+ *     .view
+ *     .view-[css_name]
+ *     .view-id-[view_name]
+ *     .view-display-id-[display_name]
+ *     .view-dom-id-[dom_id]
+ * - $classes: A string version of $classes_array for use in the class attribute
+ * - $css_name: A css-safe version of the view name.
+ * - $css_class: The user-specified classes names, if any
+ * - $header: The view header
+ * - $footer: The view footer
+ * - $rows: The results of the view query, if any
+ * - $empty: The empty text to display if the view is empty
+ * - $pager: The pager next/prev links to display, if any
+ * - $exposed: Exposed widget form/info to display
+ * - $feed_icon: Feed icon to display, if any
+ * - $more: A link to view more, if any
+ *
+ * @ingroup views_templates
+ */
+  // removes view class to avoid generic view style to interfere with styling
+  $classes = str_replace('view ','', $classes);
+?>
+
+<div class="panel-panel <?php print $classes; ?>">
+  <div class="panel-panel-inner">
+    <div class="panel-pane pane-block">
+
+      <?php if ($title): ?>
+        <h2 class="pane-title">
+          <?php print $title; ?>
+        </h2>
+      <?php endif; ?>
+
+      <div class="pane-content">
+
+        <?php if ($rows): ?>
+          <div class="view-content">
+            <?php print $rows; ?>
+          </div>
+        <?php elseif ($empty): ?>
+          <div class="view-empty">
+            <?php print $empty; ?>
+          </div>
+        <?php endif; ?>
+
+        <?php if ($footer): ?>
+          <div class="chr_panel__footer">
+            <div class="chr_actions-wrapper">
+              <?php print $footer; ?>
+            </div>
+          </div>
+        <?php endif; ?>
+
+      </div> <!-- end: pane-content -->
+    </div> <!-- end: panel-pane -->
+  </div> <!-- end: panel-panel-inner -->
+</div>  <!-- end: panel-panel -->
```

Comment:
please set up your IDE to automatically leave an empty line when saving a file

Reply:
Done.